doc/decisions/arbitrary_metadata.md

```diff
@@ -6,6 +6,8 @@ To make storage-plugins suitable for `spec` they need to be able to store
 all the metadata as specified in [METADATA.ini](/doc/METADATA.ini).
 Most file formats do not have support for that.
 
+If metadata is merged from different sources, metadata from spec might end up somewhere else.
```

Comment:
This sentence is a bit unclear to me.

Reply:
I hope I could clarify in 3c1c5bf2649d3a5e0d8ae3a61283846f30375492